dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -260,7 +260,7 @@ createLTPAKeys.option-desc.password.key=\
 #------------------------------\n at 72 chars Leading "\ \ \ \ "-------\n\#
 #------------------------------\n at 72 chars Leading "\t"-------------\n\#
 tlsProfiler.desc=\
-\tTest the target host and port's compatibility with available TLS protocols and cipher suites.
+\tTest the target host and port''s compatibility with available TLS protocols and cipher suites.
```

Comment:
I think rather than doubling what is actually an apostrophe rather than a quote, we could just remove the possessive altogether, since IBM style advises against possessives for inanimate objects/entities

 tTest the target host and port compatibility with available TLS protocols and cipher suites.